bowling.py

```diff
@@ -0,0 +1,71 @@
+with open('input.txt', 'r') as f:
+    print(sum([int(line.strip()) for line in f]))
+
+
+def read_score_of_roll(roll_num, frame_num):
+    score = int(
+        input(f"Enter score for Frame-{frame_num}, Roll-{roll_num}:  "))
+    # if score > 10:
+    #     raise "Invalid score"
+    return score
+
+
+def record_scores(list_of_scores, *values):
+    list_of_scores.append((*values,))
+
+
+def record_10_frames():
+    frame = 1
+    scores = []
+    while frame < 11:
+        score1 = read_score_of_roll(1, frame)
+
+        if frame < 10:
+            if score1 < 10:
+                score2 = read_score_of_roll(2, frame)
+                record_scores(scores, score1, score2)
+            elif score1 == 10:
+                record_scores(scores, 10)
+
+        if frame == 10:
+            score2 = read_score_of_roll(2, frame)
+            if score1 + score2 < 10:
+                record_scores(scores, score1, score2)
+                break
+            score3 = read_score_of_roll(3, frame)
+            record_scores(scores, score1, score2, score3)
+
+        frame += 1
+
```

Comment:
you could use the python range method for incrementation:  `range(start, stop, step)`
eg: 
`for frame in range(1, 11, 1)`